db/seeds/development.rb

```diff
@@ -128,6 +128,20 @@
             credential_service_providers: [SignIn::Constants::Auth::IDME, SignIn::Constants::Auth::LOGINGOV],
             service_levels: [SignIn::Constants::Auth::LOA3, SignIn::Constants::Auth::IAL2])
 
+# Create VerifiedRepresentative and AccreditedInvidial for logging into accredited_representative_portal
+ogc_registration_number = '123'
+poa_code = '678'
+test_rep_email = 'vets.gov.user+1@gmail.com'
+individual_type = 'representative'
+
+accredited_individual = AccreditedIndividual.find_or_initialize_by(registration_number: ogc_registration_number)
+accredited_individual.update!(ogc_id: SecureRandom.uuid,
+                              poa_code:,
+                              individual_type:)
```

Comment:
We should also make sure the AccreditedIndivdiual has the right email:
https://github.com/department-of-veterans-affairs/vets-api/blob/629a325a021c46f3834b9c9b87bf79e6ee7f3070/db/schema.rb#L76

Reply:
Fixed, thanks.